src/main/AuthenticationResponse.ts

```diff
@@ -256,56 +231,66 @@ function extractPresentations(resOpts: AuthenticationResponseOpts) {
   };
 }
 
-async function createSIOPResponsePayload(
-  verifiedJwt: VerifiedAuthenticationRequestWithJWT,
-  resOpts: AuthenticationResponseOpts
-): Promise<AuthenticationResponsePayload> {
+async function createSIOPIDToken(verifiedJwt: VerifiedAuthenticationRequestWithJWT, resOpts: AuthenticationResponseOpts): Promise<IdToken> {
   assertValidResponseOpts(resOpts);
   if (!verifiedJwt || !verifiedJwt.jwt) {
     throw new Error(SIOPErrors.VERIFY_BAD_PARAMS);
   }
   const supportedDidMethods = verifiedJwt.payload.registration?.subject_syntax_types_supported?.filter((sst) =>
     sst.includes(SubjectSyntaxTypesSupportedValues.DID.valueOf())
   );
-
   const state = resOpts.state || getState(verifiedJwt.payload.state);
   const nonce = verifiedJwt.payload.nonce || resOpts.nonce || getNonce(state);
-  const registration = createDiscoveryMetadataPayload(resOpts.registration);
-
-  //https://sphereon.atlassian.net/browse/VDX-140
-  // *********************************************************************************
-  // todo We are missing a wrapper object. Actually the current object is the id_token
-  // *********************************************************************************
-
-  const { verifiable_presentations, vp_token } = extractPresentations(resOpts);
-  const authenticationResponsePayload: AuthenticationResponsePayload = {
+  const id_token: IdToken = {
     iss: ResponseIss.SELF_ISSUED_V2,
-    sub: resOpts.did,
     aud: verifiedJwt.payload.redirect_uri,
-    did: resOpts.did,
-    sub_type: supportedDidMethods.length && resOpts.did ? SubjectIdentifierType.DID : SubjectIdentifierType.JKT,
-    state,
-    nonce,
     iat: Date.now() / 1000,
     exp: Date.now() / 1000 + (resOpts.expiresIn || 600),
-    registration,
+    sub: resOpts.did,
+    auth_time: Date.now() / 1000,
+    nonce,
+    _vp_token: {
+      // right now pex doesn't support presentation_submission for multiple VPs (as well as multiple VPs)
+      presentation_submission: {
+        id: uuidv4(),
+        definition_id: verifiedJwt.presentationDefinitions[0]?.definition.id || undefined,
+        descriptor_map: [
+          {
+            format:
+              resOpts.vp && ObjectUtils.isString(resOpts.vp[0]) ? VerifiablePresentationTypeFormat.JWT_VP : VerifiablePresentationTypeFormat.LDP_VP,
+            path: '$',
```

Comment:
this doesn't seem right. I know pex doesn't support multiple VPs, but we can use it to create `PresentationSubmission` object for one VP

Reply:
Since this is an optional field (according to [this](https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0-ID1.html#section-5.2-6)). I think we can remove this for now and add it back when we have the support for it in pex